### PR TITLE
[PPP-11292] Database Seeding

### PR DESCRIPTION
**Pull request by jlam-mpc**
Features:
- Added JSONs of ServiceNow data from 1/25/2023
- Parse JSON and convert to table rows with seeding file

### CODE REVIEW

File: prisma/seed.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                    synced: new Date(),
+                    created: new Date(forecast.sys_created_on),
+                    modified: new Date(forecast.u_modified),
+                    updated: new Date(forecast.sys_updated_on),
+                    active: forecast.active == 'true',
+                    featured: forecast.u_featured_opportunity =='true',
+                    // state: Number(forecast.state),
+                    
+                    estimated_value: forecast.u_est_value,
+                    // fiscal_year: Number(forecast.u_fiscal_year.slice(-2)),
```
Comment (elizabeth-jimenez) [inline prisma/seed.ts:46]: I know this is probably overkill, at this stage, but we could eventually also make sure that there are 2 digits to slice

```
            const pattern = /.*\d{2,}$/i;
            const regex = new RegExp(pattern);
            const match = forecast.u_fiscal_year.match(regex);
            const fiscal_year = match ? Number(forecast.u_fiscal_year.slice(-2)) : null;
```

Comment (jlam-mpc) [reply prisma/seed.ts:46]: I actually just added a TODO note in that file about checking what inputs we're getting. Good call- I'll be doing this for MVP

Comment (jlam-mpc) [reply prisma/seed.ts:46]: Actually I'll just throw it in now, thanks Elizabeth

Comment (elizabeth-jimenez) [reply prisma/seed.ts:46]: 😱 our brains are in sync 🎤
No problem!

File: README.md [hunk 2]
```diff
@@ -38,6 +38,19 @@ Setup prisma (and restart development server)
...
+Once a local db is setup and connected, it is necessary to get the tables schemas created. It is also ideal to seed the database with data from ServiceNow as of 1/25/23 (provided as JSON files).
+
+1. Create Tables
+`npx prisma generate` or `npx prisma db push`
+
+2. Seed Database
+`npx prisma db seed`
+
+3. View Database
+`nps prisma studio`
```
Comment (elizabeth-jimenez) [inline README.md:52]: just a small typo
❤️  really cool and useful feature and so easy to get going

File: prisma/seed.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+const prisma = new PrismaClient()
+
+async function main() {
+    for (let i = 0; i < forecasts.length; i++) {
+        const forecast = forecasts[i];
+        if (forecast) {
+            // TODO: Check for expected patterns in other relevant fields
+
+            const expectedFY = forecast.u_fiscal_year.match(new RegExp(/.*\d{2,}$/i));
+            const expectedTAQ = forecast.u_target_award_quarter.match(new RegExp(/Q[1234]/i));
```
Comment (elizabeth-jimenez) [inline prisma/seed.ts:13]: 😻

File: README.md (hunk 2 above)
Comment (jlam-mpc) [reply README.md:52]: I never would've caught this thanks for pointing it out.

**Review by elizabeth-jimenez — APPROVED** [review]
🤌

**Review by elizabeth-jimenez — APPROVED** [review]
🤌 🤌

**Pull request merged**